src/core/lv_refr.c

```diff
@@ -899,8 +912,9 @@ void refr_obj(lv_draw_ctx_t * draw_ctx, lv_obj_t * obj)
         lv_opa_t opa = lv_obj_get_style_opa(obj, 0);
         if(opa < LV_OPA_MIN) return;
 
-        lv_area_t layer_area_full;
-        lv_res_t res = layer_get_area(draw_ctx, obj, layer_type, &layer_area_full);
+        lv_area_t layer_area_ext, layer_area_ext_trans, layer_area_full;
```

Comment:
Are you sure `layer_area_ext, layer_area_ext_trans` are really required? My view ways that only the layer area is needed as it will be rendered. The rest should comes naturally from the transformation. 

In other words the "layer module" should not know about the such a thing like an `obj_ext_size`. It just creates a buffer where LVGL can render and then blends/transforms the that buffer.

Reply:
There are sometimes 1px misalign, I'll see if I can solve that without draw the whole object.
![image](https://user-images.githubusercontent.com/830358/173517343-b409ca46-e1bc-42d4-9781-2b7937b81f70.png)